fv3core/tests/savepoint/conftest.py

```diff
@@ -61,8 +62,24 @@
     "PressureAdjustedTemperature_NonHydrostatic",
     "DynCore",
     "Tracer2D1L",
+    "CopyCorners",
     "FillCornersVector",
     "A2B_Ord4",
+    "Remapping",
+    "CS_Profile_2d",
+    "CS_Profile_2d_2",
+    "MapScalar_2d",
+    "Map1_PPM_2d",
+    "Map1_PPM_2d_2",
+    "Map1_PPM_2d_3",
+    "MapN_Tracer_2d",
+    "SatAdjust3d",
+    "Fillz",
+    "MoistCVPlusPkz_2d",
+    "MoistCVPlusPt_2d",
+    "CubedToLatLon",
+    "Neg_Adj3",
+    "FVDynamics",
```

Comment:
We have all the tests covered in dace right? If so, we can remove the concept of `CURRENT_DACE_SAVEPOINT_TESTS` now.

Reply:
Good catch